lib/filters.js

```diff
@@ -51,6 +58,17 @@ function httpHeaders (payload) {
         .map(function (k) { return k + '=' + cookies[k] })
         .join('; ')
     }
+
+    if (typeof headers['set-cookie'] !== 'undefined') {
```

Comment:
I don't think we need to protect our selfs from people overwriting the `undefined` property right? If so you can just do a simple comparison instead. Unless there's an edge case I'm not considering?

Reply:
You mean do `if (headers['set-cookie'] !== undefined)` instead?

Reply:
yep

Reply:
fixed